src/Tools/dotnet-monitor/HostBuilder/HostBuilderHelper.cs

```diff
@@ -21,7 +21,7 @@ internal static class HostBuilderHelper
 
         public static IHostBuilder CreateHostBuilder(HostBuilderSettings settings)
         {
-            string aspnetUrls = string.Empty;
+            string? aspnetUrls = string.Empty;
```

Comment:
This seems like it should be non-nullable.

Reply:
This can be null - we set it using `webBuilder.GetSetting(WebHostDefaults.ServerUrlsKey);` which may be null. If it is, we apply our own defaults.